### DISCUSSION TITLE
ODBC 连接doris时无法查看Multi catalog的数据

### DISCUSSION THREAD

**Discussion by Tinglessss** [apache/doris#40335] [RFC]
我想使用Mysql ODBC driver 连接doris，但是好像只能看到它自己的数据库，无法查看external catalog的数据，有什么解决办法吗？

Doris版本：3.0.1
MySQL ODBC driver: 9.0.0

**Comment by zy-kkk** [comment]
Currently, when connecting to Doris using the mysql protocol, you can only connect and display one catalog. If you can use SQL operations, you can use `switch catalog` to switch different catalogs, or you can use `catalog.db` as the full name of the database when connecting to specify the catalog you want to connect to.
